### ISSUE TITLE
Inconsistent availability of tensor methods for activation

### ISSUE THREAD

**Issue by ngimel** [pytorch/pytorch#41568]
Currently the following commonly used activation functions are available as tensor methods (`x.relu`): relu, prelu, hardshrink, sigmoid, softmax 
The following activations are not available as tensor methods: elu, gelu, hardtanh, tanhshrink, softmin, silu. 
Some of the latter are available as torch namespace functions, some are only available in torch.nn.functional. 
There is no system to which functions end up where, and we have to be more consistent about it. 
cc @albanD @mruberry

**Comment by vadimkantorov** [comment]
Related: https://discuss.pytorch.org/t/torch-tanh-vs-torch-nn-functional-tanh/15897/7
